guidelines/index.html

```diff
@@ -3199,7 +3199,7 @@ <h2 id="x6-glossary"><span class="secno">6. </span>用語集<span class="permali
    
    <ul>
       
-      <li><strong>一般閃光</strong>は、暗いほうの映像の<a href="#dfn-relative-luminance" class="internalDFN" data-link-type="dfn">相対輝度</a>が 0.80 未満である、最大相対輝度の 10％以上の相対輝度の相反する変化の組合せとして定義される。ここでいう「相反する変化の組合せ」とは、増加した後に減少する、又は減少した後に増加するものである。そして、
+      <li><strong>一般閃光</strong>は、暗いほうの映像の<a href="#dfn-relative-luminance" class="internalDFN" data-link-type="dfn">相対輝度</a>が 0.80 未満である、最大相対輝度 (1.0) の 10％以上の相対輝度の相反する変化の組合せとして定義される。ここでいう「相反する変化の組合せ」とは、増加した後に減少する、又は減少した後に増加するものである。そして、
```

Comment:
訳としてはあっていると思うのですが、「最大相対輝度 (1.0) の 10％以上」というのが何なのかわかりにくく感じました。ここでは「相対輝度の最大値が1.0であり、その10%以上」という意味なのですが、伝わりにくいように思います。

"the maximum relative luminance" はこの場所でしか使われていないようで、「最大相対輝度」という専門用語があるというよりも、"relative luminance" の "the maximum" の値である、ということのように思います。「相対輝度の最大値」としてはどうでしょうか。
また、原文では「暗いほうの…」は後ろにある which句になっています。語順も原文に合わせたほうがわかりやすいように感じました。

アイデアベースですが、このようにしてみてはどうでしょうか?
```suggestion
      <li><strong>一般閃光</strong>は、相対輝度の相反する変化が相対輝度の最大値 (1.0) の 10％以上となる組合せであり、暗いほうの映像の<a href="#dfn-relative-luminance" class="internalDFN" data-link-type="dfn">相対輝度</a>が 0.80 未満であるものとして定義される。ここでいう「相反する変化の組合せ」とは、増加した後に減少する、又は減少した後に増加するものである。そして、
```

Reply:
ありがとうございます。異論ありません。（修正いただいた文面によって、すっと頭に入りやすくなった印象です。）
2つめのセンテンスに、「相反する変化の組合せ」("a pair of opposing changes") という句があるので、1つめのセンテンスでもこの句は保持したいと思います。別途、コミット出します。